build/README.md

```diff
@@ -522,7 +522,7 @@ existing clusters you will need to:
 1.  Run `tk apply workspace/$CLUSTER_CONTEXT_schema_manager`
 
 ### Database version and server cache
-Current version: 3.1.0
+Current version: 3.1.1
 Minimum support version: 3.0.0
 
 #### Important: Server version should NEVER be behind Database Version ####
```

Comment:
Let's remove this note as the semantic versioning of the database means that code intended to work with v3.0.0 of the database should also work with, e.g., v3.1.1 of the database.